pkg/store/store.go

```diff
@@ -49,11 +51,19 @@ type Store interface {
 
 // Key is a key for the object store.
 type Key struct {
-	Namespace  string      `json:"namespace"`
-	APIVersion string      `json:"apiVersion"`
-	Kind       string      `json:"kind"`
-	Name       string      `json:"name"`
-	Selector   *labels.Set `json:"selector"`
+	Namespace            string         `json:"namespace"`
+	APIVersion           string         `json:"apiVersion"`
+	Kind                 string         `json:"kind"`
+	Name                 string         `json:"name"`
+	Selector             *labels.Set    `json:"selector"`
+	RequirementsSelector *[]Requirement `json:"requirementsSelector"`
+}
+
+// See vendor/k8s.io/apimachinery/pkg/labels/selector.go:120
```

Comment:
the line number will go stale over time. If the reference is useful, link to a specific commit, otherwise remove the reference.